docs/Makefile

```diff
@@ -0,0 +1,9 @@
+all: 
+	mkdir -p output
+	docker build -t sota_docs_site .
+	docker run -t -i -v $(PWD)/output:/data/output sota_docs_site jekyll build -d output
+
+clean:
+	rm -rf output
+
+.PHONY: all 
```

Comment:
Also worth adding 'clean' to the phony targets list, FYI
https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html